test/unit/org/apache/cassandra/service/reads/DigestResolverTest.java

```diff
@@ -78,6 +80,70 @@ public void noRepairNeeded()
         assertPartitionsEqual(filter(iter(response)), resolver.getData());
     }
 
+    @Test
+    public void multiThreadedNoRepairNeededReadCallback() throws InterruptedException
```

Comment:
Nit: doesn't need `throws InterruptedException`

Reply:
Nit: We could mention the ticket number in a comment, maybe in the JavaDoc for the test.